src/components/molecules/ResourceDiff/ResourceDiff.tsx

```diff
@@ -105,6 +110,14 @@ const ResourceDiff = (props: {
     readOnly: true,
   };
 
+  const confirmModalTitle = useMemo(
```

Comment:
I saw that the `confirmModalTitle` is repeated across multiple components, shouldn't we move it to the Modal instead?

edit: I understood why it's repeated, I'm wondering if we should make a custom hook for this?